cmd/proxy/actions/app_proxy.go

```diff
@@ -16,8 +17,9 @@ func addProxyRoutes(
 ) error {
 	app.GET("/", proxyHomeHandler)
 
+	dp := goget.New()
```

Comment:
A nonsense nit but maybe `gg`, I guess `dp` was download protocol?

Reply:
@robjloranger yeah dp for download protocol because the handlers don't care what the implementation is (go get, github, storage, etc) -- I think it would be really cool if in the very near future we made one download.Protocol interface that will do most of the magic for us: go get, storage, cache filling, etc.

Reply:
And by magic, I don't really mean magic in the meta programming sense lol